### PR TITLE
2328: Use svgs

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Replace .png and .jpg images with .svg along with some cleanup.

### Proposed changes

- Replace all .png and .jpg images with .svgs. Exceptions:
  - favicons and apple touch icons (as there is no full browser support yet for .svg AFAIK)
  - social media previews (they are actual images)
  - ~app-icon.png of aschaffenburg (not good enough quality to create an svg)~
- Reuse location marker by setting color and rename files to `LandingIcon`
- Remove unused images (e.g. native assets folder)
- Change file location of `licenses.json` for web

### Side effects

None I hope.

### Testing

@Docmale this replaces quite a few icons/images in the app with .svgs which might lead to problems for your testing process. All apps (integreat, malte, aschaffenburg, obdach) should probably be tested if all the icons are still looking the same.

### Resolved issues

Fixes: #2328.

---

### CODE REVIEW

**Review by sarahsporck — APPROVED** [review]
Tested in web and android emulator for IG, Malte & Aschaffenburg. Sems to work fine :)

File: native/src/components/LandingIcon.tsx
Comment (LeandraH) [inline native/src/components/LandingIcon.tsx:1]: It took me a while to understand what this image does. I think I find EasterEggImage clearer 💭

File: web/src/components/LandingIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const LandingIcon = (): ReactElement => {
+  const [counter, setCounter] = useState(0)
+
+  const increment = () => {
+    setCounter(counter + 1)
+
+    const CRASH_COUNTER_MAX = 13
+    if (counter === CRASH_COUNTER_MAX) {
+      throw new Error('This error was thrown for testing purposes.')
```
Comment (LeandraH) [inline web/src/components/LandingIcon.tsx:23]: TIL 😂

**Review by LeandraH — APPROVED** [review]
😍 Nice! Tested in Firefox and emulated iOS, each for Integreat, Malte, and Aschaffenburg

File: native/src/components/LandingIcon.tsx
Comment (steffenkleinle) [reply native/src/components/LandingIcon.tsx:1]: Hmm, I don't like `EasterEggImage` but yes, perhaps it should be more about what it does than where its used. How about `SwitchCmsUrlIcon`?

Comment (LeandraH) [reply native/src/components/LandingIcon.tsx:1]: Great idea, that does make it clearer!

File: .prettierignore [hunk 2]
```diff
@@ -3,7 +3,7 @@ native/ios/**/*
 native/vendor/**/*
 
 native/src/assets/licenses.json
-web/assets/licenses.json
+web/src/assets/licenses.json
```
[the remark this answers is not in the document]
Comment (LeandraH) [reply .prettierignore:6]: Ah cool, thanks for the explanation :)

**Pull request merged**